yarn.lock

```diff
@@ -138,20 +138,19 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@gnosis.pm/dex-contracts@^0.1.3":
+"@gnosis.pm/dex-contracts@^0.1.2":
```

Comment:
Why is the contracts dependency being downgraded?

Reply:
same question

Reply:
Not anymore :) solved